specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/examples/SearchServiceListSkillsets.json

```diff
@@ -14,6 +14,7 @@
             "skills": [
               {
                 "@odata.type": "#Microsoft.Skills.Text.EntityRecognitionSkill",
+                "name": "#1",
```

Comment:
Are these meant to be examples of explicitly-set names or implicitly-set names? If the latter, the docs for the name property don't indicate that the name will be prefaced with "#". If the former, the example should be something other than a number so the two cases aren't confused.

Reply:
This is an implicitly-set name - I'll update the docs to mention that!